cfpb/cfgov-refresh
# Fix for rate checker notifications

[Short description explaining the high-level reason for the pull request]

## Additions

-

## Removals

-

## Changes

- Modified rate checker notifcations to use the svg markup. 

## Testing

1.  Visit http://localhost:8000/owning-a-home/explore-rates/
2. Move the slider to 600.
3. Verify the notification icons is present.
4. Disable JS.
5. Verify the notification icons is present.

## Screenshots


## Notes

-

## Todos

-

## Checklist


## Testing checklist

### Browsers


### Accessibility


### Other
